.travis.yml

```diff
@@ -30,19 +30,20 @@ stages:
 
 jobs:
   include:
-    - stage: "Audit"
-      if: type = pull_request
-      script: audit-ci --low --report-type full
+    #- stage: "Audit"
```

Comment:
TODO readd?

Reply:
I am not sure, github does auditions out-of-the-box so we could speed up our CI builds by removing the auditing